### ISSUE TITLE
nextjs 14, observer

### ISSUE THREAD

**Issue by Tekutorukushi** [LegendApp/legend-state#221]
After updating to version NextJs 14.0.2, I get an error
@legendapp/state": "^2.1.2"

`Attempted import error: 'observer' is not exported from '@legendapp/state/react' (imported as 'observer').`

Code:
```
import { observer } from '@legendapp/state/react';

const ModalsContainer = observer(({ }) => {
    return (
        <div>some code<div>
    );
});
```

**Comment by jmeistrich** [comment]
Oh wow that's surprising... We have updated to 14 yet, but we will try that just to test this soon.

**Comment by jmeistrich** [comment]
Hmm, I'm not seeing any isues after upgrading to Next 14. Do you see any other possible causes for it not working? Could you share a repo that I can test to see the problem?

**Comment by Tekutorukushi** [comment]
Git with clean NextJs
[https://gitlab.com/Tekutorukushi/nextjs](https://gitlab.com/Tekutorukushi/nextjs)

Before that I had Next 14.0.1, I updated to 14.0.2 and I got an error
Windows 11, Node v18.18.2

**Comment by jacktsin1** [comment]
I'm getting similar errors:
`Attempted import error: 'Show' is not exported from '@legendapp/state/react' (imported as 'Show').`
`Attempted import error: 'useSelector' is not exported from '@legendapp/state/react' (imported as 'useSelector').`

**Comment by Tekutorukushi** [comment]
fix in nextjs 14.0.4

**Comment by varna** [comment]
> fix in nextjs 14.0.4

Yep, fixed by 14.0.4. Ty
